src/server/accounting.c

```diff
@@ -312,6 +312,56 @@ cpy_quote_value(char *pb, char *value)
 /* start attributes */
 #define ACCTBUF_TIMES_NEED	72
 
+/**
+ * @brief
+ * Get the resources_used job attribute
+ *
+ * @param[in]	pjob	- pointer to job structure
+ * @param[in]	resc_used - pointer to resources used string
+ * @param[in]	resc_used_size - size of resources used string
+ *
+ * @return	int
+ * @retval	0 upon success
+ * @retval	-1	if error encountered.
+ *
+ */
+static int
+get_resc_used(job *pjob, char *resc_used, int resc_used_size) {
```

Comment:
Also, what's the purpose of 'resc_used_size' ? It seems like it's an output variable. Do you intend to store the size of the resources_used string here? If yes, then this should be an int pointer right?

Reply:
same comment for *resc_used, it should be **resc_used since concat_rescused_to_buffer can realloc it

Reply:
Thank you @agrawalravi90 for catching this.